widgetssdk/src/main/java/com/glia/widgets/core/fileupload/GliaFileProvider.java

```diff
@@ -2,5 +2,13 @@
 
 import androidx.core.content.FileProvider;
 
+/**
+ * Glia internal class.
+ *
+ * Will be automatically added to integrator Manifest by Manifest merger during compilation.
+ *
+ * This custom FileProvider is used to prevent confusion with request to the default FileProvider that might be used by the integrator.
```

Comment:
```suggestion
 * This custom FileProvider is used to prevent confusion with a request to the default FileProvider that might be used by the integrator.
```